README.md

```diff
@@ -14,98 +14,220 @@ Instagram - [@chaynhq](https://www.instagram.com/chaynhq/)
 
 Youtube - [Chayn Team](https://www.youtube.com/channel/UC5_1Ci2SWVjmbeH8_USm-Bg)
 
+# Bloom Backend
+
+[![Bloom Backend CI Pipeline](https://github.com/chaynHQ/bloom-backend/actions/workflows/.ci.yml/badge.svg)](https://github.com/chaynHQ/bloom-backend/actions/workflows/.ci.yml)
+
+**Currently in active development**
+
 ## Technologies Used:
 
-- Nest Js - Nest is a NodeJs framework used in building scalable and reliable server-side applications.
-- PostgreSQL - open source object-relational database system
-- TypeORM is an Object Relational Mapper library running in NodeJs and written in TypeScript.
-- Docker - open source containerization platform.
-- Heroku - Heroku is a platform as a service (PaaS) that enables developers to build, run, and operate applications entirely in the cloud.
-- GitHub Actions - GitHub Actions makes it easy to automate all your software workflows, now with world-class CI/CD.
+- [NestJS](https://nestjs.com/) - NodeJs framework for building scalable and reliable server-side applications
+- [PostgreSQL](https://www.postgresql.org/about/) - Object-relational SQL database system
+- [TypeORM](https://github.com/typeorm/typeorm) - Object Relational Mapper library
+- [Firebase](https://firebase.google.com/docs/auth) - User authentication
+- [Storyblok](https://www.storyblok.com/) - Headless CMS for pages and courses content
+- [Simplybook](https://simplybook.me/en/) - Appointment booking system used for therapy
+- [Slack](https://api.slack.com/messaging/webhooks) - Slack webhooks to send messages to the team
+- [Rollbar](https://rollbar.com/) - Error reporting
+- [Crisp](https://crisp.chat/en/) - User messaging
+- [Docker](https://www.docker.com/) - Containers for api and db
+- [Heroku](https://heroku.com) - Build, deploy and operate staging and production apps
+- [GitHub Actions](https://github.com/features/actions) - CI pipeline
 - Swagger - API documentation
-- Firebase - User Authentication
 
-## Development Setup
+## Local development
 
-- Install npm modules `yarn install`. To install [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
-- Copy `.env.example` and rename as `.env`
+### Prerequisites
 
-## Adding environmental variables
+- NodeJS v12.x
```

Comment:
Could add a sentence on the option of using `nvm` to manage node versions + install [link](https://github.com/nvm-sh/nvm#installation-and-update)?

We could also make the case to have this in the `ways of working` doc rather than here.